src/client/styles/components/_button.scss

```diff
@@ -15,12 +15,12 @@
   box-shadow: none;
   display: block;
   float: left;
-  line-height: 56px;
+  //line-height: 56px;
```

Comment:
Rather than comment out lines, you can remove them. Git keeps a history of changes.